ui/ui.js

```diff
@@ -247,8 +247,10 @@ shaka.ui.Overlay = class {
       fullScreenElement: this.videoContainer_,
     };
 
-    // Check AirPlay support
-    if (window.WebKitPlaybackTargetAvailabilityEvent) {
+    // eslint-disable-next-line no-restricted-syntax
+    if ('remote' in HTMLMediaElement.prototype) {
+      config.overflowMenuButtons.push('remote');
+    } else if (window.WebKitPlaybackTargetAvailabilityEvent) {
       config.overflowMenuButtons.push('airplay');
```

Comment:
This is necessary because we support it since Safari 13, and the API is only available since Safari 13.1. https://caniuse.com/mdn-api_remoteplayback